tensorflow/python/ops/ctc_ops.py

```diff
@@ -308,6 +314,10 @@ def ctc_greedy_decoder(inputs, sequence_length, merge_repeated=True):
     sequence_length: 1-D `int32` vector containing sequence lengths, having size
       `[batch_size]`.
     merge_repeated: Boolean.  Default: True.
+    blank_index: (optional) Set the class index to use for the blank label.
```

Comment:
(Optional).  Default: -1.  The class index to use for the blank label.

Reply:
Thank you!